--- a/vendor/keboola/php-component/src/Manifest/ManifestManager/Options/OutTable/ManifestOptions.php
+++ b/vendor/keboola/php-component/src/Manifest/ManifestManager/Options/OutTable/ManifestOptions.php
@@ -16,8 +16,13 @@ use function is_array;
 
 class ManifestOptions
 {
-    private const ALLOWED_MANIFEST_TYPES = [self::MANIFEST_TYPE_OUTPUT];
+    private const ALLOWED_MANIFEST_TYPES = [self::MANIFEST_TYPE_OUTPUT, self::MANIFEST_TYPE_OUT];
     public const MANIFEST_TYPE_OUTPUT = 'output';
+    public const MANIFEST_TYPE_OUT = 'out';
+
+    private const MANIFEST_TYPE_NORMALIZATION_MAP = [
+        self::MANIFEST_TYPE_OUT => self::MANIFEST_TYPE_OUTPUT,
+    ];
     private ?string $destination = null;
 
     private ?bool $incremental = null;
@@ -158,7 +163,7 @@ class ManifestOptions
             ));
         }
 
-        $this->manifestType = $manifestType;
+        $this->manifestType = self::MANIFEST_TYPE_NORMALIZATION_MAP[$manifestType] ?? $manifestType;
         return $this;
     }
 
